x/auth/mock/random_simulate_blocks.go

```diff
@@ -0,0 +1,47 @@
+package mock
+
+import (
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/abci/types"
+	"math/rand"
+	"strconv"
+	"testing"
+	"time"
+)
+
+// RandomizedTesting tests application by sending random messages.
+func (app *App) RandomizedTesting(t *testing.T, ops []TestAndRunTx, setups []RandSetup,
+	invariants []AssertInvariants, numKeys int, numBlocks int, blockSize int) {
+
+	time := time.Now().UnixNano()
```

Comment:
I think we want an option to set a specific seed, for reproducability, and *also* to output the seed when a random seed is used, so we can reproduce any errors we find.

Reply:
The seed is currently added to the log file, for reproducability purposes. I'll add a seperate method to take the seed as a parameter, so its easier to test if the bug is being reproduced.